doc/man/bitcoind.1

```diff
@@ -389,6 +389,14 @@ Include IP addresses in debug output (default: 0)
 .IP
 Prepend debug output with timestamp (default: 1)
 .HP
+\fB\-avoidpartialspends\fR
+.IP
+Group outputs by address, selecting all or none, instead of selecting on
+a per\-output basis. Improved privacy as an address is only used
```

Comment:
would say `Privacy is improved as an address...`

Reply:
Changed